src/vconsole/vconsole-setup.c

```diff
@@ -392,7 +392,7 @@ static int font_load_and_wait(const char *vc, Context *c) {
 /*
  * A newly allocated VT uses the font from the source VT. Here
  * we update all possibly already allocated VTs with the configured
- * font. It also allows to restart systemd-vconsole-setup.service,
+ * font. It also allows one to restart systemd-vconsole-setup.service,
```

Comment:
"It also allows systemd-vconsole-setup.service to be restarted to apply a new font to all VTs."